### PR TITLE
Added still yet even more inhand drinking sprites

### PR DESCRIPTION
**Pull request by JMWTurner**
feakijdazij

that one too
aaaaaa

Fixed a few oversights from the last PR, and added inhand sprites for all condiments.

I'm hoping to keep'em coming once per day

### CODE REVIEW

**Comment by kol1th** [comment]
_nice_

File: html/changelogs/JMWTurner-StillMoreDrinks.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: JMWTurner
+delete-after: true
+changes:
+  imageadd: Still more inhand sprites, this time mostly for condiments
```
Comment (Intigracy) [inline html/changelogs/JMWTurner-StillMoreDrinks.yml]: You left out the - and it's going to break the changelog if this is merged

Comment (JMWTurner) [reply html/changelogs/JMWTurner-StillMoreDrinks.yml]: Saw your comment right as I pushed the commit to fix it.

**Comment by CptWad** [comment]
those are not drink sprites
at least I hope nobody drinks straight up ketchup, soy sauce, flour or whatever Universal enzyme is

**Comment by JMWTurner** [comment]
@CptWad 

/obj/item/weapon/reagent_containers/food/drinks/flour

When you attack_self a condiment, the text says that you "drink" some of it. So they're pretty much drinks.

Also universal enzymes is yeast.

**Pull request merged**